ContributionGuidelines.md

```diff
@@ -0,0 +1 @@
+Don't.
```

Comment:
I think you missed couple of important facts in your first paragraph "... you dare".